the-10-min/src/Picker/GestureHandler.tsx

```diff
@@ -33,6 +34,7 @@ const GestureHandler = ({ value, max, defaultValue }: GestureHandlerProps) => {
     velocity: velocity.y,
     state,
     snapPoints,
+    offset: new Value(-ITEM_HEIGHT * 34),
```

Comment:
@wcandillon Sorry if I'm missing this, but why the number "34"? What does this represent?

Reply:
I believe that's a hardcoded default value

Reply:
I do it like this:
 
 GestureHandler.tsx
 ```
 offset: new Value(0),
 ```
 
 AnimationHelpers.tsx
 ```
const defaultIndex = findIndex(values, (v) => v.value === defaultValue);

cond(not(init), [
    set(offset, -ITEM_HEIGHT * defaultIndex),
    set(state.position, offset),
    set(init, 1),
  ]),
 ```